src/styles/blocks/catalog.scss

```diff
@@ -1,17 +1,17 @@
+@use "../utils/extends";
+@use "../utils/variable" as *;
+
 .catalog {
   display: flex;
   flex-wrap: wrap;
-
   max-width: 944px;
   margin: 0 auto;
   padding: 50px 0;
 
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
+    @extend %centre-flex;
     box-sizing: border-box;
-
+    transition: all $transition-speed ease;
```

Comment:
Don't apply transition to all the properties. It's really bad for the performance.
Please specify to which exactly properties you want to apply transition 
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x